### PR TITLE
OD-91 - Rename "Dataset" to "Data Profiles" and "Resource" to "Data"

### PR DESCRIPTION
**Pull request by sivang**
things to review:
- make sure I did not overcopy templates instead of using {% blocks %}.
- See that labels make sense.

### CODE REVIEW

**Comment by sivang** [comment]
OD-91 is the user story in Jira, linked from the first comment on this issue

> @JGulic will do a proper review probably. Meanwhile, have few general question/comments
> 
> * What does OD-91 mean?
> * It's not quite clear what the PR is about from it's description. Since you've added several new files it's almost impossible to understand what has changed. Please describe it in PR description
> * do we really need to update `-bs2` templates?
> * would be much better to extend just blocks instead of copying whole file (unless it's really necessary or >70% of lines has changed) This would make it easy to understand where exactly the change was made and therefore review

File: ckanext/ed/plugin.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def before_search(self, search_params):
     # IConfigurer
     def update_config(self, config_):
         toolkit.add_template_directory(config_, 'templates')
+        toolkit.add_template_directory(config_, 'templates-bs2')
```
Comment (zelima) [inline ckanext/ed/plugin.py]: We don't need this anymore, right?

File: ckanext/ed/templates/snippets/add_dataset.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{# Adds 'Add Dataset' button #}
+
+{% set dataset_type = dataset_type if dataset_type else 'dataset' %}
+
+{% if group %}
+    {% link_for _('Add Data Profile'), controller='package', action='new', group=group, class_='btn btn-primary', icon='plus-square' %}
+{% elif dataset_type == 'dataset' %}
```
Comment (zelima) [inline ckanext/ed/templates/snippets/add_dataset.html]: Is `dataset_type` string? Just asking, cause on line 10 it has `.title()` property

File: ckanext/ed/plugin.py (hunk 1 above)
Comment (sivang) [reply ckanext/ed/plugin.py]: Bullocks, I had overlooked it I guess.

File: ckanext/ed/templates/snippets/add_dataset.html (hunk 2 above)
Comment (sivang) [reply ckanext/ed/templates/snippets/add_dataset.html]:
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.2.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: dataset = 'add dataset'                                                                                                                                                              

In [2]: dataset.title()                                                                                                                                                                      
Out[2]: 'Add Dataset'
```

**Pull request merged**